harmoni-web-api/pom.xml

```diff
@@ -38,7 +38,11 @@
             <groupId>org.springframework.session</groupId>
             <artifactId>spring-session-core</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>com.github.librepdf</groupId>
+            <artifactId>openpdf</artifactId>
+            <version>1.3.30</version>
+        </dependency>
```

Comment:
The library used uses a viral license, for the purpose of thesis is acceptable, before commercial use you need to read the license in depth